ui/packages/catalog-ui-search/src/main/webapp/js/CQLUtils.js

```diff
@@ -351,12 +352,12 @@ function buildIntersectCQL(locationGeometry) {
 function arrayFromPolygonWkt(wkt) {
   // Handle POLYGON with no internal rings (i.e. holes)
   if (wkt.startsWith('POLYGON')) {
-    const polygon = wkt.match(/\(\([^\(\)]+\)\)/g)
+    const polygon = wkt.match(/\(\([^()]+\)\)/g)
     return polygon.length === 1 ? arrayFromPartialWkt(polygon[0]) : []
   }
 
   // Handle MULTIPOLYGON with no internal rings (i.e. holes)
-  let polygons = wkt.match(/\(\([^\(\)]+\)\)/g)
+  let polygons = wkt.match(/\(\([^()]+\)\)/g)
```

Comment:
✏️ Doesn't look like `polygons` is being re-assigned after instantiation. Would be best as `const`